lib/logitech_receiver/status.py

```diff
@@ -95,9 +96,8 @@ def __init__(self, receiver, changed_callback):
 
 	def __str__(self):
 		count = len(self._receiver)
-		return (_("No paired devices.") if count == 0 else
-				_("1 paired device.") if count == 1 else
-				(str(count) + _(" paired devices.")))
+		return (gettext.gettext("No paired devices.") if count == 0 else
```

Comment:
`_` is an alias for `gettext.gettext` (see `lib/solaar/i18n.py`), can you use that?

maybe it is an idea to export `ngettext` in i18n.py so you do not have to import gettext here.